Makefile

```diff
@@ -0,0 +1,10 @@
+# Experimental, haven't verified yet
+CXX:=clang++
+CXXFLAGS:=-Wall -Wextra -Isrc/ -std=c++17 -ggdb
+CXXLIBS:=-lSDL2 -lSDL2_ttf
+SRCFILES := $(shell find src/ -type f -name "*.cpp")
```

Comment:
Can we do this without using shell?

Reply:
`SRCFILES:=$(wildcard src/**/*.cpp)` Might work, but I can't test it at the moment.